test/trust-proxy.test.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const sget = require('simple-get').concat
+const fastify = require('..')
+
+test('trust proxy', (t) => {
+  t.plan(5)
+  const app = fastify({
+    trustProxy: true
+  })
```

Comment:
Can you add some tests for the other documented options?

Reply:
Ok. But in general it in any way constructs function that return boolean, so making lots of tests will just "retest" library that's used under the hood.
Also probably we could simplify and remove some options like comma-separated ips. Just added those in order to have maximum compatibility with express to simplify migration.